natlas-server/entrypoint.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-mkdir -p /data/{db,media}
+mkdir -p /data/{db,media} logs
+
+DATABASE_FILE=${SQLALCHEMY_DATABASE_URI//sqlite:\/\/\//}
+if [ ! -f "${DATABASE_FILE}" ]; then 
+  echo "[+] Creating database"
+  export FLASK_APP=./natlas-server.py
+  flask db upgrade
+fi
 
 chown -R www-data:www-data /data
-exec runuser -u www-data -- "$@"
+chown -R www-data:www-data logs
+exec runuser -m -p -u www-data -- "$@"
```

Comment:
`-m` and `-p` have the same meaning. https://linux.die.net/man/1/runuser

Reply:
These can be removed. It was me tinkering with it.